### PR TITLE
Pr example

### PR DESCRIPTION
**Pull request by NPIPHI**
Added the effect of solar radiation on the net force

details

link to the math

### CODE REVIEW

**Comment by gautamdayal** [comment]
Spell 'solar' correctly

File: src/PhysicsEngine.cpp [hunk 1]
```diff
@@ -335,6 +335,14 @@ Vector3d RungeKutta::calc_net_force(double tStamp, Vector3d pos_if,
     Vector3d net_force_if = rocket_.r2i(aero_force_rf + thrust_rf);
     net_force_if.z() -= (9.81 * mass);
 
+
+    //account for the pressure of the sunlight
+
+    Vector3d SolerRadiationForce{0.000000001, 0.0000002, 0.000000001};
```
Comment (ary-dz) [inline src/PhysicsEngine.cpp:341]: Soler -> Solar

File: src/PhysicsEngine.cpp [hunk 2]
```diff
@@ -335,6 +335,14 @@ Vector3d RungeKutta::calc_net_force(double tStamp, Vector3d pos_if,
...
+
+    //add solar radiation force to the net force
+    net_force_if + SolerRadiationForce
```
Comment (gautamdayal) [inline src/PhysicsEngine.cpp:344]: semicolon pepeANGRY

File: src/PhysicsEngine.cpp (hunk 1 above)
Comment (AyberkY) [inline src/PhysicsEngine.cpp:341]: nit: use scientific notation for very small numbers i.e. `1e-9`

Comment (rishivg) [reply src/PhysicsEngine.cpp:341]: missed a semicolon

**Pull request closed without merging**